### PR TITLE
rev_Add logic for collecting .log output

### PR DESCRIPTION
**Pull request by mlabuda**
- at the beginning of each test clear log
- after test execution collect logs
- store logs in a file
- log processing should be done via Eclipse API, not via UI

### CODE REVIEW

**Comment by rawagner** [comment]
I tried this locally and looks good to me but Im missing some tests which would verify at least some basic functinality

**Comment by mlabuda** [comment]
I had a class with main method locally and tried it on various examples and it worked. It is a bit difficult to write tests for this because you would have to create a new test class/test suite with test classes which would use RedDeerSuite runner and it would have to be called inside of test method with made up error/some output to log file and then evaluate it on same place. I spoke with @apodhrad  and we agreed its not necessary to have tests for it, that tests for comparator and sorting of prioritized Befores/Afters is enough. Are you ok with this?

File: plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/log/collector/LogCollector.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+			String line = br.readLine();
+			if (line != null) {
+				bw.write(logDescription + "\n\n");
+				bw.write(line + "\n");
+				while ((line = br.readLine()) != null) {
+					bw.write(line + "\n");
+				}
+				bw.write("\n\n");
+			}
+		} catch (FileNotFoundException e) {
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/log/collector/LogCollector.java]: at least print stacktrace

File: plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/log/collector/LogCollector.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+		} catch (IOException e1) {
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/log/collector/LogCollector.java]: same here

**Pull request merged**